### PR TITLE
[Ready] Rebalances Chameleon Kits for lowpop

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton ShizCalev Kor Kevinz000 (original idea)
balance: Due to budget cuts, the shoes shipped with chameleon kits no longer prevent slipping. The premium version is unchanged and still sold separately.
balance: To compensate, the chameleon kit cost was lowered by 2 TC and the minimum crew limit removed.
/:cl:

Some time ago, chameleon kits were given a minimum player limit due to the noslips that come with them.
This PR removes the noslip ability from chameleon kit shoes, reduces the kit price to 2 TC and removes the player limit. Regular noslip shoes are still available and unchanged.

That way, lowpop tators can have disguises while highpop is unaffected, having the same total price for the old chameleon+noslips combo.

Code:
I created a shoes/chameleon/noslip subtype and moved the NOSLIP_1 flag to it. Also, I made sure that all other regular noslips refer to the new subtype.

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -710,9 +710,8 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	name = "Chameleon Kit"
 	desc = "A set of items that contain chameleon technology allowing you to disguise as pretty much anything on the station, and more!"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/uplink/uplink_items.dm]: "Shoes sold separately!"

Comment (81Denton) [reply code/modules/uplink/uplink_items.dm]: Good idea

**Comment by Robustin** [comment]
Cham kits having free noslips was just lazy, stupid design.

**Comment by ExcessiveUseOfCobblestone** [comment]
It was made by Goof so fair.

File: code/game/objects/items/storage/uplink_kits.dm [hunk 2]
```diff
@@ -257,14 +257,13 @@
...
 
-/obj/item/storage/box/syndie_kit/chameleon
+/obj/item/storage/box/syndie_kit/chameleon // No longer contains noslips, but is cheap and has no player minimum
 	name = "chameleon kit"
 
 /obj/item/storage/box/syndie_kit/chameleon/PopulateContents()
 	new /obj/item/clothing/under/chameleon(src)
 	new /obj/item/clothing/suit/chameleon(src)
 	new /obj/item/clothing/gloves/chameleon(src)
-	new /obj/item/clothing/shoes/chameleon(src)
```
Comment (ShizCalev) [inline code/game/objects/items/storage/uplink_kits.dm]: Make a new subtype `/obj/item/clothing/shoes/chameleon/noslip` and move the noslip flag to that instead imo, that way you don't have to split up the kit like this. :(

Comment (81Denton) [reply code/game/objects/items/storage/uplink_kits.dm]: Yeah, that seems like the proper way compared to having an incomplete chameleon kit.
It's gonna be a while until I can fix my PR, should I leave it open or do people normally close and reopen a new one?

**Comment by 81Denton** [comment]
Going to fix this and then reopen later

**Comment by PKPenguin321** [comment]
Good change

**Pull request merged**